contributing-docs/testing/unit_tests.rst

```diff
@@ -1336,8 +1362,8 @@ you to quickly figure out the right version without knowing the root cause of th
 Assume you suspect library "foo" that was downgraded from 1.0.0 to 0.1.0 is causing the problem. Bisecting
 technique looks like follows:
 
-* enter breeze with ``--force-lowest-dependencies`` flag (the ``foo`` library is downgraded to 0.1.0). Your
-  test should fail.
+* Run ``uv sync --resolution lowest-direct``(the ``foo`` library is downgraded to 0.1.0). Your test should
```

Comment:
Very nice!!

Reply:
Yep. One of the nice things we are going to get